### PR TITLE
enhance siftool environment to use more than one witness

### PR DESCRIPTION
**Pull request by banshee**

### CODE REVIEW

**Review by Brando753 — APPROVED** [review]
I assume the watcher scripts are for the more in-depth debugging. Besides what we talked about in the huddle, this looks good to me.

File: smart-contracts/src/watcher/watcher.ts [hunk 1]
```diff
@@ -1,94 +1,107 @@
-import { filter, map } from "rxjs/operators"
-import { connectable, interval, merge, Observable, ReplaySubject, Subscription } from "rxjs"
-import { isNotNullOrUndefined, jsonParseSimple, tailFileAsObservable } from "./utilities"
-import { EbRelayerEvent, toEvmRelayerEvent } from "./ebrelayer"
-import { SifnodedEvent, toSifnodedEvent } from "./sifnoded"
-import { HardhatRuntimeEnvironment } from "hardhat/types"
-import { BridgeBank, CosmosBridge } from "../../build"
+import {filter, map} from "rxjs/operators"
```
Comment (smartyalgo) [inline smart-contracts/src/watcher/watcher.ts:1]: Nit: I think you have the wrong linter setup

**Review by smartyalgo — CHANGES_REQUESTED** [review]

File: smart-contracts/src/watcher/watcher.ts (hunk 1 above)
Comment (banshee) [reply smart-contracts/src/watcher/watcher.ts]: Done

**Comment by smartyalgo** [comment]
LGTM,

**Review by smartyalgo — APPROVED** [review]

**Pull request merged**